.yamato/promotion.yml

```diff
@@ -1,30 +1,24 @@
-test_editors:
-  - version: 2019.1
-test_platforms:
-  - name: win
-    type: Unity::VM
-    image: package-ci/win10:stable
-    flavor: b1.large
----
 promote:
   name: Promote to Production
   agent:
-    type: Unity::VM
-    image: package-ci/win10:stable
-    flavor: b1.large
+    type: Unity::VM::osx

```

Comment:
@jechter What is the reason for this change?    This script pushes the package to production, really need to make sure this script works 100%.   The mac agents tend to be very unreliable.

Reply:
Because the publish.sh script packing and uploading the samples is a mac shell script. I guess we'd have to turn it into a windows PowerShell script or something. Got any experience with that?